entrypoint_debian.sh

```diff
@@ -60,6 +60,45 @@ case ${OPENHAB_VERSION} in
         cp -av "${APPDIR}/userdata.dist/." "${APPDIR}/userdata/"
       fi
 
+      # Upgrade userdata if versions do not match
+      curVersion=$(< "${APPDIR}/userdata/etc/version.properties" grep build-no | cut -d : -f 2 | tr -d '[:space]')
+      imgVersion=$(< "${APPDIR}/userdata.dist/etc/version.properties" grep build-no | cut -d : -f 2 | tr -d '[:space]')
+      
+      if [ "${curVersion}" = "${imgVersion}" ]; then
```

Comment:
Looks like typo here — shouldn't it be `!=` instead of `=`?

Reply:
Indeed. I was testing without the ! so I don't have to actually get a different version of the image to run the script. I forgot to take it out. It has been corrected.